src/main/resources/application.conf

```diff
@@ -19,7 +19,9 @@ datastax-java-driver {
   }
   advanced.metrics {
     id-generator{
-      class = io.stargate.sgv2.jsonapi.service.cqldriver.CustomTaggingMetricIdGenerator
+      # This will add 'session' tag to session and node level metrics with the session name
+      # that we set to the tenant name. (Also adds a node tag for node level metrics)
+      class = TaggingMetricIdGenerator
```

Comment:
Which package will the class be in when left as unqualified? (I guess it has to be a built-in id generator provided by driver itself)